Configurations/Sanitizers.xcconfig

```diff
@@ -31,6 +31,9 @@ WK_SANITIZER_GCC_OPTIMIZATION_LEVEL_Debug = 0;
 WK_SANITIZER_GCC_OPTIMIZATION_LEVEL_Production = 1;
 WK_SANITIZER_GCC_OPTIMIZATION_LEVEL_Release = 1;
 
+GCC_PREPROCESSOR_DEFINITIONS = $(inherited) $(WK_ENABLE_CONJECTURE_ASSERT_$(WK_ANY_SANITIZER_ENABLED));
+WK_ENABLE_CONJECTURE_ASSERT_YES = ENABLE_CONJECTURE_ASSERT=1;
```

Comment:
```suggestion
GCC_PREPROCESSOR_DEFINITIONS = $(inherited) $(WK_ENABLE_CONJECTURE_ASSERT_$(WK_OR_$(WK_ANY_SANITIZER_ENABLED)_$(ENABLE_WEBKIT_CONJECTURE_ASSERT)));
WK_ENABLE_CONJECTURE_ASSERT_YES = ENABLE_CONJECTURE_ASSERT=1;
```

Reply:
Elliot, to clarify, the environment variable is not for including CONJECTURE_ASSERT in the build.  It is always included in the build for sanitizer and Debug builds.

Instead, the environment variable is for enabling the use of CONJECTURE_ASSERTs at runtime (if it's already included in the build) when running JSC/WebKit.  Hence, this change is not needed.